test/testtoken.cpp

```diff
@@ -355,6 +355,22 @@ class TestToken : public TestFixture {
         ASSERT_EQUALS(false, tok.isUtf32());
         ASSERT_EQUALS(false, tok.isLong());
         ASSERT_EQUALS(true, tok.isCMultiChar());
+
+        tok.str("'\''");
```

Comment:
I have the feeling this code should be :

     tok.str("'\\''");

with your code there is no backslash in the string. the mStr length should be 4